### PR TITLE
NMS-14807: Cortex plugin doc update

### PR DESCRIPTION
**Pull request by mmahacek**
### All Contributors

* [X] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [X] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-14807

### CODE REVIEW

**Comment by mmahacek** [comment]
I updated the instructions on which config files to make it closer how we reference "create or modify FILE" in other pages vs piping an echo to a file.  This also makes it clearer for how to prepare the config files for users that may be deploying with a container.

File: docs/modules/deployment/pages/time-series-storage/timeseries/cortex.adoc [hunk 1]
```diff
@@ -23,59 +23,63 @@ The minimum supported Mimir version is 2.0.0.
...
-
-config:update
+writeUrl=http://cortex-service-host:9009/api/prom/push<1>
+readUrl=http://cortex-service-host:9009/prometheus/api/v1<2>
+maxConcurrentHttpConnections=100
+writeTimeoutInMs=1000
+readTimeoutInMs=1000
+metricCacheSize=1000
+externalTagsCacheSize=1000
+bulkheadMaxWaitDurationInMs=9223372036854775807
```
Comment (dino2gnt) [inline docs/modules/deployment/pages/time-series-storage/timeseries/cortex.adoc:62]: why-not-both.gif

Comment (mmahacek) [reply docs/modules/deployment/pages/time-series-storage/timeseries/cortex.adoc]: image

File: docs/modules/deployment/pages/time-series-storage/timeseries/cortex.adoc [hunk 2]
```diff
@@ -59,23 +67,41 @@ property-set readTimeoutInMs 1000
 property-set metricCacheSize 1000
 property-set externalTagsCacheSize 1000
 property-set bulkheadMaxWaitDurationInMs 9223372036854775807
-
 config:update
 ----
 
-. Configure the plugin to remain installed after a restart of {page-component-title}:
+CFG File::
```
Comment (Bonrob2) [inline docs/modules/deployment/pages/time-series-storage/timeseries/cortex.adoc:73]:
```suggestion
CFG file::
```

**Review by Bonrob2 — APPROVED** [review]
LGTM. Thanks for updating.

**Pull request merged**